narui_core/src/vulkano_render/vk_util.rs

```diff
@@ -57,16 +52,25 @@ impl VulkanContext {
         let (device, queues) = devices_vec
             .into_iter()
             .find_map(|physical| {
-                let queue_family = physical.queue_families().map(|qf| (qf, 0.5)); // All queues have the same priority
+                let queue_family = physical.queue_families().map(QueueCreateInfo::family).collect(); // All queues have the same priority
                 let device_ext = DeviceExtensions {
                     khr_swapchain: true,
                     khr_storage_buffer_storage_class: true,
                     khr_8bit_storage: true,
                     // Comment in if you need shader printf
                     // khr_shader_non_semantic_info: true,
-                    ..(*physical.required_extensions())
+                    ..DeviceExtensions::none() //                    ..(*physical.required_extensions())
```

Comment:
cleanup

Reply:
right I was not sure if this is the proper way now, do we not need some extensions from the physical device?

Reply:
required extensions are now requested automatically so it is safe to just delete that line